README.md

```diff
@@ -75,6 +75,7 @@ If you cannot provide a valid website for your company your PR will not be accep
 - [SPI Nepal](http://www.softvision.com/) | Thapathali, Kathmandu | Provider of Digital Solutions, Mobility, Analytics, Merchandise Planning, In-Store Engagements and Quality Assurance
 - [SUBISU CABLENET (P.) LTD.](http://subisu.net.np/) | Jhamshikhel, Lalitpur | Internet, Network and Television Service Provider
 - [Three Monks](http://www.threemonk.com/) | Sinamangal, Kathmandu | Carter the growing needs in the space of technical expertise with special focus on Digital Financial Services, Money Transfer Business and Entertainment Companies
+- [Techrise](http://www.techrise.me/) | Jhamshikhel, Lalitpur | We are an innovative Learning Platform combining 1-on-1 mentor training, project-based learning, and a friendly student community that will accelerate your path to becoming a web developer 
```

Comment:
Hello @rabingaire, `Techrise` should be above `Three Monks` alphabetically. Can you also shorten your description a bit? It should be not more than 2 lines while [viewing](https://github.com/rabingaire/tech-companies-in-nepal/blob/f1a7ac59371b2d754d98a08af1f46f90c4da7575/README.md).